source/integrations/cloud-bot-accounts.rst

```diff
@@ -82,7 +82,7 @@ Frequently Asked Questions
 Should I migrate all my integrations to use bot accounts?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-For your integrations using RESTful API and plugins, yes. To do so, you can either convert an existing account to a bot, or create a new bot account using the steps outlined above.
+For your integrations using RESTful API and plugins, yes. To do so, you can either convert an existing account to a bot or create a new bot account using the steps outlined above.
```

Comment:
The pre-existing comma preceding ``or`` is correct since it separates two independent clauses.

Reply:
ok